### DISCUSSION TITLE
Obtaining full file system zip file?

### DISCUSSION THREAD

**Discussion by WKirklandJr** [abrignoni/iLEAPP#613] [Q&A] [answered]
Is there a reference for obtaining the full file system of an iPhone to utilize within iLEAPP? I tried using the zip file of an iPhone backup however it didnt' process any information from it.

**Comment by abrignoni** [comment, marked as answer]
A phone backup is not a full file system extraction. To create a full file system extraction you need to have a jailbroken iPhone.  This blogpost explains how the process would look like. https://abrignoni.blogspot.com/2018/08/finding-discord-chats-in-ios.html

How to jailbreak a device and extract a full file system from it will change from phone to phone and some might not even have a jailbreak available. 

Another way to generate a full file system extraction is to user third-party tooling. Sadly these are not available to regular consumers. 

In regards to backups, iLEAPP parses iTunes backups as long as they are not encrypted. 

There is no need or reason to compress the backup using zip. For details see this video: 
https://youtu.be/l6z9UrjAR9c?feature=shared

**Reply by WKirklandJr** [reply]
Thanks for the reply and the link to the video! I somehow missed the part in the description that iLEAPP can parse a iTunes/Finder Backup Folder.
